### DISCUSSION TITLE
Feature Request: Force the plugin GUI on server side to remain open when launched with Crossover 21

### DISCUSSION THREAD

**Discussion by CrossGridder** [apohl79/audiogridder#803] [Ideas]
Thanks a lot for your effort on this project. Based on my researches documented in this discussion [https://github.com/apohl79/audiogridder/discussions/797](url) I'd like to request if it could be possible to maintain the plugin window on the server side always open and on top when using with Crossover 21. Basically, I can see the UI on the server for one second (or less) only when disabling screen capture and local mode. Sandboxing and parallel plugin need to be on, if not the server will keep on returning the port binding error. That way it will be possible to use vsts that are only available for windows. I am willing to sponsor you with a crossover 21 license for you to consider to add this feature. Latency and resources usage are a fraction compared to a vm environment and possibly the best solution ever for people that doesn't have additional machines to convert as servers.

**Comment by CrossGridder** [comment]


**Reply by apohl79** [reply]
Hey, thanks for staying on top of this! I'm a little side tracked right now, thats why I'm a bit slow in responding these days.. Just share what you did. And if there is a nice addition or helpful feature to make it more usable, I'm happy to add a feature request and implement it at some point.

**Comment by Teebone2000** [comment]
Amazing!

    On Friday, 19 November 2021, 22:58:54 GMT, Federico Braina ***@***.***> wrote:  
 
 


I will make a proper tutorial soon. First I need to recover from 4 days of basically non-stop testing. There is one more positive thing that I didn't take into account before. People with 2 macs can have a windows server without having to use bootcamp or vm.


—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Henri75** [comment]
Interesting option to use crossover !
Just installed it to give it a try. AG is installing and launching, but most plugin installer doesnt install (seems to be related to missing/unregistered DLL for installer like ISSKIN.dll, which needs MFC42.dll or others).
Did you encounter that and how did you fix it ?
Or may be you installed the bottle with specific options (other than  latest microsoft visual C++ redist and .net framework) ?

**Reply by Henri75** [reply]
Got it, you need Visual C++ 6 redist for most things to work.
How did you handle the greyed AG icon in mac top menu bar which you cant right click !? I am unable to access AG pop up menu on iuts tray icon
